src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
Yes I get that, the accessibility state element has a default value of "component title Incomplete". aria-live only reads out when the content of the element changes. To make this read out in both scenarios (a correct or incorrect answer), I think maybe this would need to be changed to only contain the state after the user has provided an answer of some description. Certainly some additional thought needs to go in to it.

We should raise that as a separate accessibility improvement.